modules/pel/peltool/peltool.py

```diff
@@ -571,6 +572,8 @@ def main():
                         dest='IDToDelete', help='Delete a PEL based on its ID')
     parser.add_argument('-D', '--delete-all', dest='deleteAll',
                         action='store_true', help='Delete all PELs')
+    parser.add_argument('--archive',
```

Comment:
I suggest we also add a short option name -A.

As Matt mentioned, we should support this option only in the BMC environment, as we already have the -p option to specify the path in non-BMC environments.

Therefore, you could move this argument addition into the if condition at line 577.

```
if not inBMC:
...
else:
... # BMC specific things could be added.
```

Reply:
Done